condition/asset.go

```diff
@@ -0,0 +1,34 @@
+package condition
+
+import (
+	"net"
+
+	"github.com/google/uuid"
+	"github.com/metal-toolbox/rivets/types"
+)
+
+// Asset holds attributes of a server retrieved from the inventory store.
+//
+// nolint:govet // fieldalignment struct is easier to read in the current format
+type Asset struct {
```

Comment:
Why use this and not the `Server` type?

Reply:
I was trying to avoid import cycle issues, but we might be okay with this change.
Pushed changes to move to use the `Server` type instead, lets see how that goes.